tests/testCompoundCoordinateTransformations.py

```diff
@@ -0,0 +1,281 @@
+
+        http://aa.usno.navy.mil/data/docs/geocentric.php
+        """
+
+        hours = np.radians(360.0/24.0)
+        minutes = hours/60.0
+        seconds = minutes/60.0
+
+        longitude_list = []
+        latitude_list = []
+        mjd_list = []
+        alt_list = []
+        az_list = []
+        ra_app_list = []
+        dec_app_list = []
+
+        longitude_list.append(np.radians(-22.0-33.0/60.0))
+        latitude_list.append(np.radians(11.0+45.0/60.0))
+        mjd_list.append(2457364.958333-2400000.5) # 8 December 2015 11:00 UTC
+        alt_list.append(np.radians(41.1))
+        az_list.append(np.radians(134.7))
+        ra_app_list.append(16.0*hours + 59.0*minutes + 16.665*seconds)
+        dec_app_list.append(np.radians(-22.0 - 42.0/60.0 - 2.94/3600.0))
+
+        longitude_list.append(np.radians(-22.0-33.0/60.0))
+        latitude_list.append(np.radians(11.0+45.0/60.0))
+        mjd_list.append(2457368.958333-2400000.5) # 12 Decemer 2015 11:00 UTC
+        alt_list.append(np.radians(40.5))
+        az_list.append(np.radians(134.7))
+        ra_app_list.append(17.0*hours + 16.0*minutes +51.649*seconds)
+        dec_app_list.append(np.radians(-23.0-3/60.0-50.35/3600.0))
+
+        longitude_list.append(np.radians(145.0 + 23.0/60.0))
+        latitude_list.append(np.radians(-64.0-5.0/60.0))
+        mjd_list.append(2456727.583333-2400000.5) # 11 March 2014, 02:00 UTC
+        alt_list.append(np.radians(29.5))
+        az_list.append(np.radians(8.2))
+        ra_app_list.append(23.0*hours + 24.0*minutes + 46.634*seconds)
+        dec_app_list.append(np.radians(-3.0-47.0/60.0 - 47.81/3600.0))
+
+        longitude_list.append(np.radians(145.0 + 23.0/60.0))
+        latitude_list.append(np.radians(-64.0-5.0/60.0))
+        mjd_list.append(2456731.583333-2400000.5) # 15 March 2014, 02:00 UTC
+        alt_list.append(np.radians(28.0))
+        az_list.append(np.radians(7.8))
+        ra_app_list.append(23.0*hours + 39.0*minutes + 27.695*seconds)
+        dec_app_list.append(np.radians(-2.0 - 13.0/60.0 - 18.32/3600.0))
+
+        for longitude, latitude, mjd, alt, az, ra_app, dec_app in \
+            zip(longitude_list, latitude_list, mjd_list, alt_list, az_list,
+                ra_app_list, dec_app_list):
+
+            obs = utils.ObservationMetaData(site=utils.Site(longitude=longitude,
+                                                            latitude=latitude),
+                                            mjd=utils.ModifiedJulianDate(UTC=mjd))
+
+
+            ra_icrs, dec_icrs = utils._raDecFromAltAz(alt, az, obs)
+            ra_test, dec_test = utils._appGeoFromICRS(np.array([ra_icrs]), np.array([dec_icrs]),
+                                                      mjd=obs.mjd)
+
+            distance = np.degrees(utils.haversine(ra_app, dec_app, ra_test[0], dec_test[0]))
+            self.assertLess(distance, 0.1) # since that is all the precision we have in alt az
```

Comment:
Sorry.  That was confusing.

The 0.1 is the accuracy in Alt, Az that we have in the data I downloaded from NOAO.  I will clarify the comment.